checkov/openapi/checks/resource/v2/OperationObjectSecurityScopeUndefined.py

```diff
@@ -50,6 +50,9 @@ def scan_openapi_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[Che
                             return CheckResult.FAILED, conf
                         if not isinstance(auth_definition, dict):
                             return CheckResult.UNKNOWN, conf
+                        definition_type = auth_definition.get('type', {})
+                        if definition_type != "oauth2":
```

Comment:
add a failing state for the UTs as well

Reply:
it already exists `openapi/checks/resource/v2/example_OperationObjectSecurityScopeUndefined/fail1.json` this fix handles false positives.